### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by netlork**
es.lua
Spanish translation.
Traducción al español.

### CODE REVIEW

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
Hola, gracias por la traducción.. pero hay que cambiar a couple of things... they sound a bit weird.

**Comment by MonkeyWhisper** [comment]
["realestate_only"] = "Solo la inmobiliaria puede usar este comando",
This should read something like "Solo bienes raíces suede vender casas" 

["failed_invasion"] = "No se pudo intentar de nuevo",
La invasion de la policia fallo or something along those lines

["open_stash"] = "Abrir armario",

["stash"] = "Armario",

And for these two... armario? I'm not sure. Can we find another word for it? Maybe reservas or inventario privado?

**Comment by netlork** [comment]
ok, i check it. I'll try to match more with the context.

**Comment by MonkeyWhisper** [comment]
> ok, i check it. I'll try to match more with the context.

Yeah exactly! Okay, you understand where I'm going with it, awesome.

**Comment by netlork** [comment]
Changed. 
My only doubt is with "failed_invasion", if I'm not mistaken, is failing an attempt to force the door?

File: locales/es.lua [hunk 1]
```diff
@@ -12,10 +12,10 @@ local Translations = {
         ["not_owner"] = "No eres dueño de esta casa.",
         ["no_police"] = "No hay policía presente...",
         ["already_open"] = "Esta casa ya está abierta..",
-        ["failed_invasion"] = "No se pudo intentar de nuevo",
+        ["failed_invasion"] = "Intento fallido, prueba de nuevo.",
         ["inprogress_invasion"] = "Alguien ya está trabajando en la puerta..",
         ["no_invasion"] = "Esta puerta no está abierta...",
-        ["realestate_only"] = "Solo la inmobiliaria puede usar este comando",
+        ["realestate_only"] = "Solo la inmobiliaria puede vender casas",
```
Comment (MonkeyWhisper) [inline locales/es.lua:18]: Change the word inmobiliaria to bienes raíces - this is a more commonly known Spanish over.

File: locales/es.lua [hunk 2]
```diff
@@ -49,8 +49,8 @@ local Translations = {
         ["view_house"] = "Ver Casa",
         ["ring_door"] = "Tocar timbre",
         ["exit_door"] = "Salir de la propiedad",
-        ["open_stash"] = "Abrir armario",
-        ["stash"] = "Armario",
+        ["open_stash"] = "Abrir alijo",
```
Comment (MonkeyWhisper) [inline locales/es.lua:52]: That's not a known universally known Spanish word

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

**Comment by MonkeyWhisper** [comment]
> Changed.
> My only doubt is with "failed_invasion", if I'm not mistaken, is failing an attempt to force the door?

Yes when the police is trying to battery ram the house

File: locales/es.lua (hunk 1 above)
Comment (netlork) [reply locales/es.lua:18]: I'm sorry, but "bienes raíces" is not a common word (although it exists), the correct word is "inmobiliaria" (a business that sells houses, garages, etc).
I'm from Spain and Spanish is my native language :P

But if you consider that it is better to use "bienes raíces" I have no problem.  :)

File: locales/es.lua (hunk 2 above)
Comment (netlork) [reply locales/es.lua:52]: I can change it for "escondite" or "escondrijo", a secret and safely storage.

Comment (MonkeyWhisper) [reply locales/es.lua:52]: Where are you from? Seems like those are words from European Spanish.

I'm fine with Escondite yes, thanks

Comment (netlork) [reply locales/es.lua:52]: > Where are you from? Seems like those are words from European Spanish.
> 
> I'm fine with Escondite yes, thanks

I'm from Spain :)   So yes, those are European Spanish words.
I'll try to use language known as "neutral spanish". :)

**Review by jay-fivem — CHANGES_REQUESTED** [review]
Can you please add the new locales from #141

File: locales/es.lua (hunk 1 above)
Comment (MonkeyWhisper) [reply locales/es.lua:18]: It is bienes raices, no one know what a inmobiliaria is... lol Let's use that and I'll approve this RP 

Thank you for working with me on this

File: locales/es.lua (hunk 2 above)
Comment (MonkeyWhisper) [reply locales/es.lua]: Okay, that's good! Escondite works

**Comment by MonkeyWhisper** [comment]
> 

Of course my boi! We got you.

File: locales/es.lua (hunk 1 above)
Comment (jacanales) [reply locales/es.lua:18]: I also think "inmobiliaria" is the world use un standard spanish. "Bienes raíces" is the think that is sold, when we're talking about t'he company which sells those "bienes raíces"

**Review by jacanales — APPROVED** [review]

File: locales/es.lua (hunk 1 above)
Comment (netlork) [reply locales/es.lua]: > It is bienes raices, no one know what a inmobiliaria is... lol Let's use that and I'll approve this RP
> 
> Thank you for working with me on this.
> 
> I'm Cuban so Spanish is also my native language

Ok, it's already changed.
Although I think it's still wrong, it's not the usual way of using it and it sounds "weird".
But the important thing is that the translation is available and understandable for everyone :)

**Review by MonkeyWhisper — APPROVED** [review]
Approved, thanks.

**Comment by MonkeyWhisper** [comment]
Thank you for the corrections!

**Comment by netlork** [comment]
> Thank you for the corrections!

Thanks to you.
I'll try to translate more modules.

**Pull request merged**